src/v/redpanda/admin/api-doc/cluster.json

```diff
@@ -227,6 +227,13 @@
                     },
                     "description": "ids of all nodes being recognized as down"
                 },
+                "data_disk_degraded": {
+                    "type": "array",
+                    "items": {
+                        "type": "int"
+                    },
+                    "description": "ids of all nodes with data disk degraded"
```

Comment:
```suggestion
                    "description": "IDs of all brokers with data disk degraded"
```
To standardize with use of "broker" (instead of "node") in our public docs. 

With that said it might be nice to mention somewhere (I'm not sure how the command is invoked that would return this list of brokers, and is it just one portion of the output?) what the next steps would be, if we don't already do that - e.g. check logs, shut cluster down, etc